packages/client/src/__tests__/integration/errors/missing-binary/test.ts

```diff
@@ -10,17 +14,18 @@ test('missing-binary', async () => {
   const { PrismaClient } = require('./node_modules/@prisma/client')
 
   const platform = await getPlatform()
-  const binaryPath = process.env.PRISMA_FORCE_NAPI
-    ? path.join(
-        __dirname,
-        'node_modules/.prisma/client',
-        getNodeAPIName(platform, 'fs'),
-      )
-    : path.join(
-        __dirname,
-        'node_modules/.prisma/client',
-        `query-engine-${platform}`,
-      )
+  const binaryPath =
+    getClientEngineType() === ClientEngineType.Library
+      ? path.join(
+          __dirname,
+          'node_modules/.prisma/client',
+          getNodeAPIName(platform, 'fs'),
+        )
+      : path.join(
+          __dirname,
+          'node_modules/.prisma/client',
+          `query-engine-${platform}`,
+        )
```

Comment:
Should we maybe move this to somewhere else and just import it here vs. re-create this in each test?